CommunityFeatures.md

```diff
@@ -1,40 +1,76 @@
-
-    A documentation of the feature, how it can be used and configured shall be put in this position below every entry 
-
-
-# Runtime settings aka Community Features Menu
-
-In the main menu of the deluge (Shift + Pressing selection knob) there is an entry called "Community Features" that allows changing behavior and turning features on and off in comprison to the original and previous community firmwares. Here is a list of all options and what they do:
-
-* Force 7SEG Mode
-
-    Enable or disable, shows 7SEG rendering on the OLED instead of the OLED screen. Mainly intended for debugging purposes. Currently a dream and not really implemented
-
-# Compiletime settings
-
-This list includes all preprocessor switches that can alter firmware behaviour at compile time and thus require a different firmware
-
-* HAVE_OLED
-
-    Currently determines if the built firmware is intended for OLED or 7SEG hardware
-
-* FEATURE_...
-
-    Description of said feature, first new feature please replace this
\ No newline at end of file
+# Introduction
+
+Every time a Pull request improves the community firmware it shall note down it's achievements and usage in this document.
+
+# File Compatibility Warning
+In general, we try to maintain file compatibility with the official firmware. However, **files (including songs, presets, etc.) that use community features may not ever load correctly on the official firmware again**. Make sure to back up your SD card!
+
+# General improvements
+
+Here is a list of general improvements that have been made ordered from newest to oldest:
+
+* PR [#17] - Increase the resolution of "patch cables" between mod sources and destinations.
+* PR [#29] - Distinguish between MPE and non-MPE kit rows.
+* PR [#47] - Extra MIDI ports on the USB interface for MPE. This improves the usability of MPE-capable devices through the USB interface by providing more virtual cables.
+
+
+# Added features
+
+Here is a list of features that have been added to the firmware as a list ordered from newest to oldest:
+
+## Synthesizer features
+
+### New LFO shapes
+LFO types added to the "LFO SHAPE" shortcut.
+
+ - ([#32]) Random Walk. Starts at zero and walks up or down by small increments when triggered.
+ - ([#32]) Sample&Hold. Picks a new random value every time it is triggered.
+
+### New LFO Synchronization Modes
+Synchronization modes accessible through the "LFO SYNC" shortcut.
+
+ - ([#32]) Triplets. Synchronizes the LFO to triplet (3/2) divisions.
+ - ([#32]) Dotted. Synchronizes the LFO to "dotted" (2/3) divisions.
+
+### Filters
+ - ([#103]) adds a new filter in the low-pass slot, a state-variable filter.
+
+## New behaviors
+
+### Instrument Keyboard View
+ - ([#46]) Note offset between rows is now configurable by holding shift and using the horizontal encoder. This allows e.g. an isomorphic keyboard layout by setting the row offset to 12. The setting is saved per clip in the song file.
```

Comment:
@bfredl let me know if there's anything else to note here.

Reply:
I think the keyboard is isomorphic in any case, perhaps "e.g. an octave keyboard layout by setting the row offset to 12"